src/threema.d.ts

```diff
@@ -245,6 +245,7 @@ declare namespace threema {
         members: string[];
         administrator: string;
         access: GroupReceiverAccess;
+        createdAt: string;
```

Comment:
This should be optional (`createdAt?`), since older apps don't send it, right?

Reply:
yes!